bob-apps/bin/bobc.rs

```diff
@@ -78,54 +372,102 @@ async fn get(key: Vec<u8>, addr: Uri) {
         Ok(res) => {
             let res: tonic::Response<_> = res;
             let data: &Blob = res.get_ref();
-            info!("OK");
             info!("response meta: {:?})", res.metadata());
             info!("data len: {}, meta: {:?}", data.data.len(), data.meta);
+            let file = File::create(filename).await;
+            match file {
+                Ok(mut file) => match file.write_all(&data.data).await {
+                    Ok(()) => {}
+                    Err(e) => {
+                        error!("{:?}", e);
+                    }
+                },
+                Err(err) => {
+                    error!("{:?}", err);
+                }
+            }
         }
         Err(res) => {
-            info!("ERR");
+            error!("{:?}", res);
+        }
+    }
+}
+
+async fn exist(keys: Vec<Vec<u8>>, addr: Uri) {
+    let mut client = BobApiClient::connect(addr).await.unwrap();
+
+    let keys = keys.into_iter().map(|key| BlobKey { key }).collect();
+    let message = ExistRequest {
+        keys,
+        options: None,
+    };
+    let request = Request::new(message);
+    let res = client.exist(request).await;
+    match res {
+        Ok(res) => {
```

Comment:
For success it is better to print one key at a line in format: `<key>: <true|false>`

Reply:
Done: [eda6196](https://github.com/qoollo/bob/pull/562/commits/eda6196444d62621a945784518d4230088266869)

Reply:
Done: [eda6196](https://github.com/qoollo/bob/pull/562/commits/eda6196444d62621a945784518d4230088266869)